magento/pwa-studio
# [PWA-2552] Change default sort on search page to relevance

## Description

Change sort on search results to `relevancy` / `Best Match` by default
"Position" is removed as a sort option on the search results page
Category pages still default to listing products by `position`

## Related Issue

Closes PWA-2552

Closes PWA-2572

## Acceptance

### Verification Stakeholders

Project maintainers

### Specification




## Verification Steps

Products on search results are sorted by `relevancy` / "Best Match" by default
"Position" is removed as a sort option on the search results page
Category pages still default to listing products by `position`

#### Test scenario(s) for direct fix/feature

1. Execute a search
2. Verify that products are sorted by "Best Match"
3. Verify that "Position" is not an option in the Sort dropdown
4. Navigate to a category page
5. Verify that products are sorted by "Position" by default
6. Execute another search
7. Ensure the page loads successfully and that products are sorted by "Best Match"
8. Execute search which list categories in filters like search for "Tops Dress"

#### Test scenario(s) for any existing impacted features/areas

1. Verify that new and existing Jest tests pass
2. Verify that Cypress test passes

#### Test scenario(s) for any Magento Backend Supported Configurations

N/A

#### Is Browser/Device testing needed?

I don't think it's required, but nice to have. 

#### Any ad-hoc/edge case scenarios that need to be considered?

No

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)

No breaking change

## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
